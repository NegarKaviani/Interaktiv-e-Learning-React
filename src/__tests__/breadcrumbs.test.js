import { render, screen } from '@testing-library/react'
import Breadcrumbs from '../components/breadcrumbs'; 

test("Example 1 renders successfully", () => {
    render(<Breadcrumbs/>);

    const element = screen.getByText(/Home Page/i);

    expect(element).toBeInTheDocument();
})