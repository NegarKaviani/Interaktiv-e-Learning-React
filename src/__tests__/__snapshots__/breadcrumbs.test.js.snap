// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="breadcrumbs-wrapper bg-white h-7 flex mt-8 mb-6"
>
  <div
    className="breadcrumbs flex items-center pl-4 w-10/12"
  >
    <p
      className="text-xs mr-3 text-gray-700"
    >
      Home Page
    </p>
    <p
      className="text-xs mr-3 text-gray-300"
    >
       
      &gt;
       
    </p>
    <p
      className="text-xs text-gray-700"
    >
      e-Lerning Courses
    </p>
  </div>
  <div
    className="user bg-neutral-500 w-2/12 flex pl-3 text-xs text-white items-center"
  >
     admin 
  </div>
</div>
`;
